docs/tutorials/8a_train_sqil_sac.ipynb

```diff
@@ -0,0 +1,160 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import datasets\n",
+    "from imitation.data import huggingface_utils\n",
+    "\n",
+    "# Download some expert trajectories from the HuggingFace Datasets Hub.\n",
+    "dataset = datasets.load_dataset(\"HumanCompatibleAI/ppo-seals-HalfCheetah-v0\")\n",
+    "\n",
+    "# Convert the dataset to a format usable by the imitation library.\n",
+    "expert_trajectories = huggingface_utils.TrajectoryDatasetSequence(dataset[\"train\"])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's quickly check if the expert is any good."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from imitation.data import rollout\n",
+    "\n",
+    "trajectory_stats = rollout.rollout_stats(expert_trajectories)\n",
+    "\n",
+    "print(\n",
+    "    f\"We have {trajectory_stats['n_traj']} trajectories. \"\n",
+    "    f\"The average length of each trajectory is {trajectory_stats['len_mean']}. \"\n",
+    "    f\"The average return of each trajectory is {trajectory_stats['return_mean']}.\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "After we collected our expert trajectories, it's time to set up our behavior cloning algorithm."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from imitation.algorithms import sqil\n",
+    "from imitation.util.util import make_vec_env\n",
+    "import numpy as np\n",
+    "from stable_baselines3 import sac\n",
+    "import seals  # noqa: F401  # needed to load \"seals/\" environments\n",
+    "\n",
+    "SEED = 42\n",
+    "\n",
+    "venv = make_vec_env(\n",
+    "    \"seals/HalfCheetah-v0\",\n",
+    "    rng=np.random.default_rng(seed=SEED),\n",
+    ")\n",
+    "\n",
+    "sqil_trainer = sqil.SQIL(\n",
+    "    venv=venv,\n",
+    "    demonstrations=expert_trajectories,\n",
+    "    policy=\"MlpPolicy\",\n",
+    "    rl_algo_class=sac.SAC,\n",
+    "    rl_kwargs=dict(seed=SEED),\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As you can see the untrained policy only gets poor rewards (< 0):"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from stable_baselines3.common.evaluation import evaluate_policy\n",
+    "\n",
+    "reward_before_training, _ = evaluate_policy(sqil_trainer.policy, venv, 100)\n",
+    "print(f\"Reward before training: {reward_before_training}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "After training, we can observe that agent is quite improved (> 1000), although it does not reach the expert performance in this case."
```

Comment:
If you have time to do more tuning, great, but not a priority; this is enough to illustrate the algorithm.